c7n/resources/rds.py

```diff
@@ -637,6 +637,50 @@ def copy_restore_info(self, client, instance):
             Tags=tags)
 
 
+@actions.register('set-snapshot-copy-tags')
+class CopySnapshotTags(BaseAction):
+    """Enables copying tags from rds instance to snapshot
+
+        .. code-block: yaml
+
+            policies:
+              - name: enable-rds-snapshot-tags
+                resource: rds
+                filters:
+                  - type: value
+                    key: Engine
+                    value: aurora
+                    op: eq
+                actions:
+                  - type: set-snapshot-copy-tags
+                    enable: True
+    """
+
+    schema = type_schema(
+        'set-snapshot-copy-tags',
+        enable={'type': 'boolean'})
+    permissions = ('rds:ModifyDBInstances',)
+
+    def process(self, resources):
+        with self.executor_factory(max_workers=3) as w:
```

Comment:
I'd recommend dialing this down to 2, rds tends to have a lower threshold on rate limits

Reply:
Ah, gotcha. I didn't know RDS had that much of a lower threshold, figured 3 would be alright based on the other actions.